.github/workflows/renew_tls_certificates.yml

```diff
@@ -0,0 +1,33 @@
+name: Renew TLS Certificates
+
+on:
+  schedule:
+# Every Monday at 17:00 UTC, 09:00 PST (-08:00)
+    - cron: '0 17 * * 1'
+
+jobs:
+  run_certbot_renew:
+    name: Run certbot renew on production
+    runs-on: ubuntu-latest
+    environment: production
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Configure ssh
+        run: |
+          mkdir -p ~/.ssh/
+          echo "$SSH_KEY" > ~/.ssh/production.key
+          chmod 600 ~/.ssh/production.key
+          cat >>~/.ssh/config <<END
+          Host production
+            HostName $SSH_HOST
+            User $SSH_USER
+            IdentityFile ~/.ssh/production.key
+          END
+          ssh-keyscan -H $SSH_HOST >> ~/.ssh/known_hosts
+      - name: Renew certificates and redeploy
+        run: |
+          ssh production docker service rm rhizone-lms_nginx
+          ssh production certbot renew
+          echo "$GITHUB_TOKEN" | ssh production docker login ghcr.io --username ${{ github.actor }} --password-stdin
+          cat docker-compose.production.yml | ssh production docker stack deploy --with-registry-auth --compose-file - rhizone-lms
```

Comment:
In the event that a new version of a service was built and tagged `latest` since the last deploy, the new version will get deployed. If that new version requires a database migration to have been run, it will put the app into an inconsistent state.

Is it an option to run the `deploy.yml` workflow after renewal completes?